src/main/java/com/mjbmjb/cf/taskmaster/taskmaster/controller/TaskController.java

```diff
@@ -15,6 +15,11 @@
 
```

Comment:
if you add `@Autowired` her you don't need the `TaskController(S3Client......` stuff